pages/manage/productions/index.vue

```diff
@@ -215,53 +224,23 @@
 </script>
 
 <style lang="scss">
-  .productions-container {
-    display: flex;
-    width: 100%;
-    font-weight: 400;
-    margin-bottom: 1rem;
-
-    .table {
-      font-family: 'Open Sans', Helvetica, Arial, sans-serif;
-      font-size: 15px;
-      background-color: none;
-
-      thead {
-        background-color: $white;
-      }
-
-      tbody {
-        margin-top: 0.5rem;
-        background-color: $white;
-      }
-
-      tr {
-        display: inline-flex;
-        width: 100%;
-      }
-
-      th {
-        width: 100%;
-      }
+  .productions-index-page {
+    .productions-events-container {
+      display: flex;
+      width: 100%;
+      font-weight: 400;
+      margin-bottom: 1rem;
 
-      td {
-        width: 100%;
+      h2 {
+        font-size: 1rem;
+        font-weight: 700;
       }
     }
 
-    a {
-      font-weight: 400;
-      color: $udb-blue;
+    .panel-footer {
```

Comment:
Same suggestions as for `panel` to create a separate component for this.

Reply:
Will be implemented in next PR